### DISCUSSION TITLE
Identifying enhancer elements-sc multiomics

### DISCUSSION THREAD

**Discussion by bhekmat** [stuart-lab/signac#1477] [General]
Hi, Do you have any suggestions on how to identify enhancer elements from single cell multiomics RNA and ATAC sequencing? 

Thank you in advance.

**Comment by timoast** [comment]
Can you elaborate on what specifically you want to do? Are you aiming to distinguish between enhancers and other regulatory elements like promoters, insulators, or silencers? Or you want to call peaks on your data?

**Reply by bhekmat** [reply]
I already called peaks on my data and linked DEGs to the DARs within each cluster between my two conditions. But, I am interested in the first part of your question (distinguishing the regulatory elements). Thank you.

**Reply by timoast** [reply]
You could intersect your peaks with the CCREs from ENCODE (https://screen.encodeproject.org/) if you have human or mouse data. Otherwise, you could classify as distal/proximal based on distance to the closest TSS. If you have other types of data you can look at epigenomic marks that are characteristically found in enhancers. Ultimately you'd need to experimentally test whether an element can enhance transcription to determine if it functions as an enhancer.
